### ISSUE TITLE
hey?  is there any documentation of this? anyway to build? 

### ISSUE THREAD

**Issue by hl1x** [Sam54123/pseudo-pbr-generator#1]
you havent really said how this works

**Comment by Igrium** [comment]
Yeah... it's something I threw together in like a day that never really worked. If you want to play with it, look at how Gradle and JavaFX applications are built in general.

In theory, to run, open a command prompt and type `./gradlew run`
